packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "10.5.1",
+  "version": "10.6.0-1",
```

Comment:
What is with the dash here?

Reply:
It means that it is a [prerelease version](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string). I did this so that I could [publish](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v10.6.0-1) a version for testing in `vets-website` since `yarn link` still doesn't work due to the different yarn versions.